src/bidiMapper/domains/context/context.ts

```diff
@@ -80,6 +87,23 @@ export class Context {
     };
   }
 
+  public async navigate(
+    url: string,
+    wait: BrowsingContext.ReadinessState = 'none'
```

Comment:
A default parameter makes sense at the API level, but for clarity, I'd prefer to avoid them in internal/implementation classes.

Reply:
done